src/program/lwaftr/run/run.lua

```diff
@@ -122,7 +122,7 @@ function run(args)
 
    engine.busywait = true
    if opts.duration then
-      engine.main({duration=opts.duration, report={showlinks=true}})
+      engine.main({duration=opts.duration, report={showlinks=true, showapps=true}})
    else
       engine.main({report={showlinks=true}})
```

Comment:
I think the thing is that we need to print this information out periodically.  Both total loss and loss rate, both lifetime and over the last 5 seconds or so.  Otherwise it's less useful.  And do I actually understand correctly that this doesn't work with vmdq=false?  But we are exhausted and should schedule this for another day.